astropy/nddata/tests/test_ccddata.py

```diff
@@ -879,6 +879,34 @@ def test_write_read_multiextensionfits_custom_ext_names(ccd_data, tmpdir):
                                   ccd_after.uncertainty.array)
 
 
+def test_read_old_style_multiextensionfits(tmpdir):
+    # Regression test for https://github.com/astropy/ccdproc/issues/664
+    #
+    # Prior to astropy 3.1 there was no uncertainty type saved
+    # in the multiextension fits files generated by CCDData
+    # because the uncertainty had to be StandardDevUncertainty.
+    #
+    # Current version should be able to read those in.
+    #
+    size = 100
```

Comment:
Is 100x100 array really necessary? If the content does not matter, why not do `np.arange(4).reshape((2, 2))` so you can save on memory and avoid a call to random generator? (Sorry, should have attached it to my previous review.)

Reply:
Thanks for the suggestion, fixed.